addons/skin.estuary/xml/DialogFullScreenInfo.xml

```diff
@@ -18,69 +18,72 @@
+				<animation effect="slide" end="0,-80" time="150" condition="PVR.IsTimeShift">conditional</animation>
 				<control type="image">
-					<left>400</left>
-					<top>-240</top>
-					<right>20</right>
-					<height>350</height>
-					<texture>dialogs/dialog-bg-nobo.png</texture>
+					<depth>DepthOSD+</depth>
+					<left>10</left>
+					<top>-490</top>
+					<width>400</width>
+					<height>600</height>
+					<aspectratio aligny="bottom">keep</aspectratio>
+					<texture fallback="DefaultVideo.png" background="true">$VAR[NowPlayingPosterVar]</texture>
 					<bordertexture border="21">overlays/shadow.png</bordertexture>
 					<bordersize>20</bordersize>
+					<include>OpenClose_Left</include>
 				</control>
-				<control type="textbox">
-					<left>443</left>
-					<top>-207</top>
-					<right>80</right>
-					<height>290</height>
-					<label fallback="416">$VAR[FullScreenInfoTextBoxVar]</label>
-					<align>justify</align>
-					<autoscroll delay="5000" repeat="7500" time="5000"></autoscroll>
-				</control>
-				<control type="grouplist" id="5550">
-					<right>20</right>
-					<top>-310</top>
-					<width>1000</width>
-					<height>100</height>
-					<align>right</align>
-					<orientation>horizontal</orientation>
-					<itemgap>-10</itemgap>
-					<onup>Close</onup>
-					<ondown>Close</ondown>
-					<usecontrolcoords>true</usecontrolcoords>
-					<control type="button" id="5553">
-						<width>auto</width>
-						<height>100</height>
-						<textoffsetx>80</textoffsetx>
-						<label>$LOCALIZE[207]</label>
+				<control type="group">
+					<include>OpenClose_Right</include>
+					<control type="image">
+						<left>400</left>
+						<top>-240</top>
+						<right>20</right>
+						<height>350</height>
+						<texture>dialogs/dialog-bg-nobo.png</texture>
+						<bordertexture border="21">overlays/shadow.png</bordertexture>
+						<bordersize>20</bordersize>
 					</control>
-					<control type="button" id="5554">
-						<width>auto</width>
-						<height>100</height>
-						<textoffsetx>80</textoffsetx>
-						<onclick>Close</onclick>
-						<label>$LOCALIZE[206]</label>
-						<visible>!String.IsEmpty(VideoPlayer.CastAndRole)</visible>
+					<control type="textbox">
+						<left>443</left>
+						<top>-207</top>
+						<right>80</right>
+						<height>290</height>
+						<label fallback="416">$VAR[FullScreenInfoTextBoxVar]</label>
+						<align>justify</align>
+						<autoscroll delay="5000" repeat="7500" time="5000"></autoscroll>
 					</control>
-					<control type="button" id="5555">
-						<width>auto</width>
+					<control type="grouplist" id="5550">
+						<right>20</right>
+						<top>-310</top>
+						<width>1000</width>
 						<height>100</height>
-						<textoffsetx>80</textoffsetx>
-						<label>$LOCALIZE[19031]</label>
-						<visible>VideoPlayer.Content(livetv)</visible>
+						<align>right</align>
+						<orientation>horizontal</orientation>
+						<itemgap>-10</itemgap>
+						<onup>Close</onup>
+						<ondown>Close</ondown>
```

Comment:
Dialog.Close(fullscreeninfo) would be nicer imo

(fwiw, "close" is deprecated: https://github.com/xbmc/xbmc/blob/master/xbmc/input/ButtonTranslator.cpp#L116)